front50-sql/src/main/kotlin/com/netflix/spinnaker/front50/model/SqlStorageService.kt

```diff
@@ -19,17 +19,18 @@ package com.netflix.spinnaker.front50.model
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.netflix.spectator.api.Registry
 import com.netflix.spinnaker.front50.exception.NotFoundException
-import com.netflix.spinnaker.front50.model.ObjectType.APPLICATION
-import com.netflix.spinnaker.front50.model.ObjectType.APPLICATION_PERMISSION
-import com.netflix.spinnaker.front50.model.ObjectType.DELIVERY
-import com.netflix.spinnaker.front50.model.ObjectType.ENTITY_TAGS
-import com.netflix.spinnaker.front50.model.ObjectType.NOTIFICATION
+import com.netflix.spinnaker.front50.model.ObjectType.PROJECT
```

Comment:
interesting re-sort of the imports here - guessing that was spotless being.. helpful?

Reply:
Yes... spotless.  I will sort this lexically though, that'll be better.